cmake/MacroHelper.cmake

```diff
@@ -176,12 +176,19 @@ macro(nrn_create_file_list list_name prefix)
 endmacro()
 
 # =============================================================================
-# Copy file from source to destination in clobber mode (i.e. no overwrite)
+# Copy file from source to destination in noclobber mode (i.e. no overwrite)
 # =============================================================================
 macro(nrn_copy_file_without_overwrite source destination)
   execute_process(COMMAND cp -n ${source} ${destination})
 endmacro()
 
+# =============================================================================
+# Copy file from source to destination only if different
+# =============================================================================
+macro(nrn_copy_file_if_different source destination)
+  configure_file(${source} ${destination} COPYONLY)
+endmacro()
```

Comment:
Just nitpick: if we are just using standard configure_file behaviour without extra processing, we don’t need to create wrapper macro ?

Reply:
I wondered about that myself but decided this had a bit more clarity. I tried several approaches to getting true copy_if_different behavior but only configure_file(...COPYONLY) seems to actually do it correctly (in my opinion).